server/src/modules/maps/maps.service.ts

```diff
@@ -410,6 +411,24 @@ export class MapsService {
     }
     //#endregion
 
+    async getZones(mapID: number): Promise<MapTrackDto[]> {
+        const map = await this.mapRepo.get(mapID, { stats: true });
+
+        if (!map) throw new NotFoundException('Map not found');
+
+        const tracks = await this.mapRepo.getMapTracks(mapID);
+
+        // This is dumb but it's what the old api does
+        // \server\src\models\map.js Line 499
+        // TODO: When map sessions are done this should be removed
+
+        await this.mapRepo.updateMapStats(mapID, {
+            plays: ((map as any).stats as MapStats).plays + 1 ?? 1
```

Comment:
Should use [increment](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#atomic-number-operations) here i.e.
```ts
plays: { increment: 1 }
```
 Prisma schema defaults `plays` to 0 so we don't have to worry about it being undefined.